src/main/java/UseCaseClasses/ScheduleComparer.java

```diff
@@ -8,7 +8,7 @@
 public class ScheduleComparer {
 
     /** Compares two Schedules and returns a Entities.Schedule with entries null if both user1 and user2 do not have
-     * events during that time, and busy if user1 or user2 have events during that time.
+     * Entities.events during that time, and busy if user1 or user2 have Entities.events during that time.
```

Comment:
When I refactored the events package it changed it in the java docs so I'll have to edit that out